test/ExpressiveInstallerTest/InstallerTestCase.php

```diff
@@ -24,79 +26,111 @@
 class InstallerTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
I'm going to mark this as abstract when I merge, as the intention is for test cases to extend it.